Anlab.Jobs.MoneyMovement/Anlab.Jobs.MoneyMovement.csproj

```diff
@@ -1,18 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk;Microsoft.NET.Sdk.Publish">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <ApplicationIcon />
     <Win32Resource />
     <UserSecretsId>f3c999a5-2304-44e0-a49e-f43333bf9ccf</UserSecretsId>
     <AssemblyVersion>1.0.0.3</AssemblyVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="2.0.2" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="2.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="2.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="2.0.0" />
-    <PackageReference Include="Serilog" Version="2.10.0" />

```

Comment:
These were removed because they are in the jobs.core ?

Reply:
Yeah. I was going to move some of the config init stuff to jobs.core like it is in our newer projects, but then decided to leave good enough alone.

Reply:
Should they be put back then?